### DISCUSSION TITLE
Update_value(values) can't update more rows/columns than the original table

### DISCUSSION THREAD

**Discussion by tertris-me** [Akascape/CTkTable#43] [Q&A] [answered]
I am expecting a [["x", "z"]] table to update to [["a", "b", "c", "d"]] (adding a column). However it can only be updated to [["a", "b"]]

This is my code:
```
class App(customtkinter.CTk):
    def __init__(self):
        super().__init__()
        starting_value = [["x", "z"]]
        self.info = CTkTable.CTkTable(self, values=starting_value)
        self.info.grid(row=0, column=0)

        self.change = customtkinter.CTkButton(self, text="update to a,b,c,d", command=self.change_callback)
        self.change.grid(row=1, column=0)

    def change_callback(self):
        self.info.update_values([["a", "b", "c", "d"]])

root = App()
root.mainloop()
```

`self.info.update_values([["a", "b"],["c", "d"]])`(adding a row) is giving the same result too:
![image](https://github.com/Akascape/CTkTable/assets/127453231/aca1203c-051e-45e4-beb1-60af4905a80f)

Appreciate in any help!

**Comment by Akascape** [comment, marked as answer]
@tertris-me 
To add a new row/column, you need to first use `add_row` or `add_column` method.

**Reply by tertris-me** [reply]
I see, thanks for this lovely extension :)
